client/convert/protobuf.go

```diff
@@ -444,7 +444,7 @@ func TransactionToMessage(t flow.Transaction) (*entities.Transaction, error) {
 
 func MessageToTransaction(m *entities.Transaction) (flow.Transaction, error) {
 	if m == nil {
-		return flow.Transaction{}, ErrEmptyMessage
+		return *flow.NewTransaction(), ErrEmptyMessage
```

Comment:
Why the change from struct literals to a constructor? Is that still needed now that the `TransactionDomainTag` field is removed?

Reply:
If possible, I would change back just to keep this PR minimal.